src/control/keyboard_config.cpp

```diff
@@ -64,20 +66,6 @@ KeyboardConfig::KeyboardConfig() :
   m_keymap[SDLK_F1]        = {0, Control::CHEAT_MENU};
   m_keymap[SDLK_F2]        = {0, Control::DEBUG_MENU};
   m_keymap[SDLK_BACKSPACE] = {0, Control::REMOVE};
-
-  m_keymap[SDLK_a]         = {1, Control::LEFT};
-  m_keymap[SDLK_d]         = {1, Control::RIGHT};
-  m_keymap[SDLK_w]         = {1, Control::UP};
-  m_keymap[SDLK_s]         = {1, Control::DOWN};
-  m_keymap[SDLK_e]         = {1, Control::JUMP};
-  m_keymap[SDLK_q]         = {1, Control::ACTION};
-
-  m_keymap[SDLK_j]         = {2, Control::LEFT};
-  m_keymap[SDLK_l]         = {2, Control::RIGHT};
-  m_keymap[SDLK_i]         = {2, Control::UP};
-  m_keymap[SDLK_k]         = {2, Control::DOWN};
-  m_keymap[SDLK_o]         = {2, Control::JUMP};
-  m_keymap[SDLK_u]         = {2, Control::ACTION};
```

Comment:
Don't think there's any reason to remove the defaults for players 2 and 3. Perhaps put them back in and set a default control for the Item Pocket?

Reply:
> Ok I think I remember why i did this
> Player 1 keybinds are literally incompatible with the rest - just picture it yourself
> You and your friends would be playing a game of twister on your keyboard